### PR TITLE
stats endpoint finished

### PR DESCRIPTION
**Pull request by marianodo**
This issue is complete

### CODE REVIEW

File: app/api/stats.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+from flask import jsonify, request, g, abort, url_for, current_app
+from .. import db
+from ..models import Stop
+from . import api
+from sqlalchemy import func 
+
+
+@api.route('/stats')
+def get_stats():
+	stops = db.session.query(func.min(Stop.stop_lat) \
```
Comment (avilaton) [inline app/api/stats.py]: The result of this query is not a set of stops so maybe the name is confusing. Please change it to `result` or something like that.

Comment (marianodo) [inline app/api/stats.py]: Changed it!

File: app/api/stats.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+		,func.min(Stop.stop_lon),func.max(Stop.stop_lat),func.max(Stop.stop_lon),func.count()).first()
+	list = {'minLat': stops[0], 'minLon': stops[1], 'maxLat':stops[2],'maxLon' :stops[3],'numbers':stops[4]}
```
Comment (avilaton) [inline app/api/stats.py]: I would call this *data* instead of list.
Also bellow you should pass *data* to jsonify.

File: app/api/stats.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+from sqlalchemy import func 
+
+
+@api.route('/stats')
+def get_stats():
+	result = db.session.query(func.min(Stop.stop_lat) \
+		,func.min(Stop.stop_lon),func.max(Stop.stop_lat),func.max(Stop.stop_lon),func.count()).first()
+	data = {'minLat': result[0], 'minLon': result[1], 'maxLat':result[2],'maxLon' :result[3],'numbers':result[4]}
+    
+	return jsonify(stops=data)
```
Comment (avilaton) [inline app/api/stats.py]: here it should be just `return jsonify(data)`. I will fix that after merging.
Thanks @marianodo!

Comment (avilaton) [inline app/api/stats.py]: NOW I GET IT, sorry! I will change it anyway for `jsonify({"stops": data})`.

**Pull request merged**